terraform/staging/.secrets.auto.tfvars.example

```diff
@@ -1,7 +1,12 @@
-# arn:aws:iam::960119293461:user/terraform
-access_key  = ""
-secret_key  = ""
-region      = ""
+# terraform-user credentials from AWS Admin GovCloud
+aws_access_key_govcloud = ""
+aws_secret_key_govcloud = ""
+# aws_access_key_commercial = ""
+# aws_secret_key_commercial = ""
+
 # federalist-staging-deployer-circle
 cf_user     = ""
-cf_password = ""
\ No newline at end of file
+cf_password = ""
+
+# currently dave's personal API Key, the only way to do this...
```

Comment:
this is only for the bootstrap user, correct?

Reply:
no, the CF creds are needed for staging and production